.github/CODEOWNERS

```diff
@@ -68,8 +68,8 @@ rush.json       @calebmshafer @pmconne @wgoehrig
 /common/**/geometry-core/                         @bbastings @EarlinLutz @mgooding @RBBentley
 /common/**/hypermodeling-frontend.*               @imodeljs/display @bbastings
 /common/**/hypermodeling-frontend/                @imodeljs/display @bbastings
-/common/**/imodel-bridge.*                        @imodeljs/connectors
-/common/**/imodel-bridge/                         @imodeljs/connectors
+/common/**/itwin-connector-framework.*            @imodeljs/connectors

```

Comment:
do we want to do `itwin-connector-framework` or `itwin-connector-fwk` to shorten it a bit?

Reply:
No. Jeff's conclusion is to keep "framework".

Reply:
okay, fine by me.